### DISCUSSION TITLE
Ability to only include specific directories

### DISCUSSION THREAD

**Discussion by samiashi** [rome/tools#4512] [Suggestions]
Just like the [files.ignore](https://docs.rome.tools/configuration/#filesignore) but an option to only run on specific directories

**Comment by ematipico** [comment]
Could share more information about your proposal? Is there something you *can't* with the current configuration? What are the current limitations? Can you share real use case scenarios?

**Reply by samiashi** [reply]
I work on a monolithic repository and would like the JS formatter & linter to only run on a couple directories rather than the whole code base when running `npx rome format . --write` for example

so something like this in `rome.json`
```json
{
  ....
  "files": {
      "only": ["frontend/", "media/"]
  },
  ....
}
```

**Reply by ematipico** [reply]
Thank you for your feedback! There's some a part that hasn't been answered: can't you use `files.ignore` to achieve that? Could you explain the current limitations? 

I don't want to put any friction to your proposal; I want to ensure we will add a new configuration that solves real problems.

**Reply by samiashi** [reply]
Hi, apologies on the late reply.

The problem with this is I'm using a monorepo with 10s of files & folders and I want rome to only ever run on one specific directory for example. 
Using `files.ignore` would technically work but would mean I'll have to maintain this array with all the folders/files that are not needed for this tool rather than just saying I want rome to run on the following.

Hope this clears things up
